docker/start-temporal.sh

```diff
@@ -3,13 +3,14 @@
 set -eu -o pipefail
 
 : "${SERVICES:=}"
-: "${SERVICE_FLAGS:=}"
```

Comment:
This is minor breaking change. I doubt anyone use it and do support it. Would mention this in release notes though.

Reply:
sure